neptune/new/internal/backends/offline_neptune_backend.py

```diff
@@ -19,7 +19,8 @@
 
 from neptune.new.exceptions import NeptuneOfflineModeFetchException
 from neptune.new.internal.backends.api_model import Attribute, FloatAttribute, StringAttribute, \
```

Comment:
For long import lists it's good to [wrap all imported modules in parentheses](https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses).

I also suggest to keep every item in separate line sorted alphabetically.